helm-charts/explorer/templates/explorer-configmap.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-config
+  labels:
+    name: {{ $.Values.db.name }}
+    app.kubernetes.io/name: {{ $.Values.db.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}  
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }} 
+data:
+
+  basic-network.json: |-
+      {
+      "name": "basic-network",
+      "version": "1.0.0",
+      "client": {
+        "tlsEnable": true,
+        "adminCredential": {
+          "id": "{{ .Values.explorer.username }}",
+          "password": "{{ .Values.explorer.password }}"
+        },
+        "enableAuthentication": true,
+        "organization": "org1MSP",
```

Comment:
Why Orgname is hardcoded here?

Reply:
made Orgname as configurable in values.yaml